src/mmw/apps/geoprocessing_api/management/commands/exportrequestcsv.py

```diff
@@ -0,0 +1,28 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+
+class Command(BaseCommand):
+    """
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument('limit', type=int)
+
+    def handle(self, *args, **options):
+        self.stdout.write('Randomly marking inlets...')
```

Comment:
This feels like it doesn't belong here ...

Reply:
Oh that was me messing around with something on another branch. Sorry.